github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -71,6 +71,29 @@ periodics:
       image: quay.io/kubevirtci/flakefinder:v20220505-8aeca1a5
       name: ""
       resources: {}
+- annotations:
+    testgrid-create-test-group: "false"
+  cluster: ibm-prow-jobs
+  cron: 25 1 * * *
+  decorate: true
+  extra_refs:
+  - org: kubevirt
+    repo: project-infra
+    base_ref: main
+    work_dir: true
+  name: periodic-push-kubevirt-flakefinder-report-values
+  spec:
+    containers:
+    - args:
+      - |
+        go run robots/cmd/push-flakefinder-results/main.go --pushgateway-url=http://pushgateway
+      command: [ "/usr/local/bin/runner.sh", "/bin/bash", "-ce" ]
+      env:
+      - name: GIMME_GO_VERSION
+        value: "1.16"
+      image: quay.io/kubevirtci/golang:v20220728-1410a63
```

Comment:
I think there is a later golang image - `v20220906-4f60dd3`

Reply:
Updated.